core/invoker/src/main/scala/whisk/core/containerpool/docker/DockerContainerFactory.scala

```diff
@@ -41,6 +41,7 @@ class DockerContainerFactory(config: WhiskConfig, instance: InstanceId, paramete
   /** Initialize container clients */
   implicit val docker = new DockerClientWithFileAccess()(ec)
   implicit val runc = new RuncClient(ec)
+  implicit val useRunc = config.invokerUseRunc
```

Comment:
Please remove this implicit as well.

Reply:
done.  sorry; had removed that at some point yesterday but lost the change.